Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -3355,4 +3355,19 @@ final class RecoveryTests: ParserTestCase {
       ]
     )
   }
+  
+  func testRecovery185() {
+    // <rdar://1181099123>
+    assertParse(
+      """
+      test { (1️⃣[X]) in }
```

Comment:
Do we have a test case for missing label but with colon? e.g. `test { (: FooBar) in }`

Reply:
With an attribute would also be interesting, ie. `test { (foo @bar baz) in }`.

Also note that all the `translated/*` tests were translated from the compiler parser tests. It'd probably be better to have this one appropriately named in `ExpressionTests` (though in an ideal world we'd split that file up too).

Reply:
> Do we have a test case for missing label but with colon? e.g. `test { (: FooBar) in }`

For some reason I deleted that test case from the final commit, I'll add it back.